students/juyeob/westagram/users/views.py

```diff
@@ -57,16 +57,25 @@ def post(self, request):
             email    = data['email']
             password = data['password']
 
+            user = User.objects.get(email = email)
+
             if not User.objects.filter(email = email).exists():
                 return JsonResponse({"message": "INVALID_USER"}, status=401)         
 
-            if not User.objects.filter(password = password).exists():    
-                return JsonResponse({"message": "INVALID_USER"}, status=401)
+            if not bcrypt.checkpw(password.encode('utf-8'), user.password.encode('utf-8')):
+                    return JsonResponse({"message": "INVALID_PASSWORD"}, status=401)
+
+            access_token = jwt.encode({ 'user_id' : user.id }, settings.SECRET_KEY, algorithm='HS256').decode('utf-8')
+
+            return JsonResponse({
+                "message" : "SUCCESS",
```

Comment:
성공의 결과로 token을 보내기 때문에 매번 메시지를 보낼 필요는 없습니다.
```suggestion
                "message" : "SUCCESS",
```